runtime/src/legacy/mod.rs

```diff
@@ -58,8 +58,8 @@ impl Runtime for Legacy {
         &self,
         request: Request<StartRequest>,
     ) -> Result<Response<StartResponse>, Status> {
-        let port = 8001;
-        let address = SocketAddr::new(Ipv4Addr::LOCALHOST.into(), port);
+        let service_port = 8001;
```

Comment:
I wasn't very smart with these port numbers. Sorry :see_no_evil: 

So I think we have 4 to dance around at this point:
1. 8001 which is the control plane in deployer
2. 8000 which is the proxy in deployer
3. The random port deployer (used to) start the service on
4. 8001 which I had here again. Maybe just move this to 6001 or 7001

(For local I would start provisioner on 5001)